deegree-services/deegree-webservices-handbook/src/main/asciidoc/restapi.adoc

```diff
@@ -9,10 +9,19 @@ workspaces or resources and start a different workspace.
 
 The servlet that handles the REST interface is already running if you
 use the standard _web.xml_ deployment descriptor. For security reasons
-the REST API is only accessible after successful authentication against
-the servlet container. When using Apache Tomcat you'll need to add a
-user with the role _deegree_ to your Tomcat configuration
-_conf/tomcat-users.xml_ file.
+the REST API is secured by default with an API key read from the 
+_config.apikey_ file in deegree workspace directory.
+
+The API key can be provide in multiple different ways.
+* As header value of key `X-API-Key`
+* As authorization header of bearer type
+* As basic authorization password where username will be ignored
+* As parameter `token`
+* As parameter `api_key`
+
+TIP: If there is no _config.apikey_ file, one will be generated on startup
+with an random value. A value of `*` will turn it off but we strongly advise
```

Comment:
Consider to make the message more explicit: 

Alternatively, a value of `*` in _config.apikey_ will turn off security for the REST API. We strongly advise against doing this in productive environments.

Reply:
This was applied in https://github.com/deegree/deegree3/pull/1498/commits/6fb2b92838a0d1981bbfaaad5f0cf7437e5219c1